templates/repo/home.tmpl

```diff
@@ -5,18 +5,14 @@
 		{{template "base/alert" .}}
 		{{template "repo/code/recently_pushed_new_branches" .}}
 		{{if and (not .HideRepoInfo) (not .IsBlame)}}
-		<div class="repo-description">
-			<div id="repo-desc" class="gt-word-break tw-text-16">
-				{{$description := .Repository.DescriptionHTML $.Context}}
-				{{if $description}}<span class="description">{{$description | RenderCodeBlock}}</span>{{else if .IsRepositoryAdmin}}<span class="no-description text-italic">{{ctx.Locale.Tr "repo.no_desc"}}</span>{{end}}
-				<a class="link" href="{{.Repository.Website}}">{{.Repository.Website}}</a>
-			</div>
-			<form class="ignore-dirty" action="{{.RepoLink}}/search" method="get">
-				<div class="ui small action input">
-					<input name="q" value="{{.Keyword}}" placeholder="{{ctx.Locale.Tr "search.code_kind"}}">
-					{{template "shared/search/button"}}
-				</div>
-			</form>
+		<div class="repo-description gt-word-break">
+			{{- $description := .Repository.DescriptionHTML ctx -}}
+			{{- if $description -}}
+				{{- $description | RenderCodeBlock -}}
+			{{- else if .IsRepositoryAdmin -}}
+				<span class="no-description text italic">{{ctx.Locale.Tr "repo.no_desc"}}</span>
```

Comment:
```suggestion
				<span class="no-description">{{ctx.Locale.Tr "repo.no_desc"}}</span>
```

Imho, I prefer no italic here and the previous class was broken a long time.

Reply:
It is the same as:

![image](https://github.com/go-gitea/gitea/assets/2114189/e701dd8e-0100-484c-9eb5-d81ca2faad87)

Reply:
```suggestion
				<span class="no-description tw-italic">{{ctx.Locale.Tr "repo.no_desc"}}</span>
```